src/Dropdown/Dropdown.tsx

```diff
@@ -52,13 +60,14 @@ export const Dropdown: FC<Props> = ({
         <Select
           id={id}
           disabled={disabled}
-          defaultValue={list.length === 1 ? String(list[0].value) : placeholder}
-          onChange={(e: React.FormEvent<HTMLSelectElement>) =>
-            onSelect(e.currentTarget.value)
-          }
+          onChange={(e: React.FormEvent<HTMLSelectElement>) => {
+            setValue(e.currentTarget.value);
+            onSelect(e.currentTarget.value);
```

Comment:
since those two lines are logically identical to the [45-46], can we create a function, with a single argument that calls ***seValue*** and ***onSelect*** ?

Reply:
Yep, will do